pkg/gds/emails/emails.go

```diff
@@ -47,27 +47,32 @@ type VerifyContactData struct {
 // VerifyContactURL composes the link to verify the contact from the context. If the
 // link is not able to be composed, the function returns an empty string and logs an
 // error because without the link the email is fairly useless.
-func (d VerifyContactData) VerifyContactURL() string {
+func (d VerifyContactData) VerifyContactURL() *url.URL {
```

Comment:
Changing this to a URL will fix future problems with URLs and ensure proper encoding … we should use `url.URL` in all of our templates instead of `string`.